src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.Dao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private Dao<Bet> betDao = new BetDaoImpl();
+    private Dao<User> userDao = new UserDaoImpl();
```

Comment:
```suggestion
    private Dao<Bet> betDao = new BetDaoImpl();
    private Dao<User> userDao = new UserDaoImpl();
```
Geberic types will erase during runtime. So you will have to links of type Dao. This will became a problem when we will start using annotation. 
Let's have two different interfaces(and they can extend this one)